Dynamic/Types/SalutationType.php

```diff
@@ -48,8 +49,9 @@ public function build(FormBuilderInterface $builder, FormField $field, string $l
     protected function getChoices(): array
     {
         return [
-            'sulu_form.salutation_mr' => 'mr',
             'sulu_form.salutation_ms' => 'ms',
+            'sulu_form.salutation_mr' => 'mr',
+            'sulu_form.salutation_other' => 'other',
```

Comment:
I think this is not the correct way of handling this. As a `Salutation` is never gender neutral (as I know no gender neutral friendly form). The salutation is used in `Letters` and something like `Hello Other Alexander Schranz,` is I think not the expected result.

The correct way for me would be here make sure that there is no default and no `Please choose`, something like `No salutation` or `No choice` / `Keine auswahl` should be the default value of the form.

Reply:
Did just now see this is the `FormBundle` this salutations represent `Salutation` of ContactBundle and so this need to be changed there first.